packages/fast-usdc/src/cli/operator-commands.js

```diff
@@ -56,21 +72,58 @@ export const addOperatorCommands = program => {
         offer,
       };
 
-      outputActionAndHint(bridgeAction, {
-        stderr: process.stderr,
-        stdout: process.stdout,
-      });
+      outputActionAndHint(bridgeAction, { stderr, stdout });
     });
 
   operator
     .command('attest')
     .description('Attest to an observed Fast USDC transfer')
     .requiredOption('--previousOfferId <string>', 'Offer id', String)
-    .action(async options => {
-      const { previousOfferId } = options;
-      console.error(
-        'TODO: Implement attest logic for request:',
+    .requiredOption('--forwardingChannel <string>', 'Channel id', String)
+    .requiredOption('--recipientAddress <string>', 'bech32 address', String)
+    .requiredOption('--blockHash <0xhex>', 'hex hash', parseHex)
+    .requiredOption('--blockNumber <number>', 'number', parseNat)
+    .requiredOption('--blockTimestamp <number>', 'number', parseNat)
+    .requiredOption('--chainId <string>', 'chain id', Number)
+    .requiredOption('--amount <number>', 'number', parseNat)
+    .requiredOption('--forwardingAddress <string>', 'bech32 address', String)
+    .requiredOption('--txHash <0xhexo>', 'hex hash', parseHex)
+    .option('--offerId <string>', 'Offer id', String, `operatorAttest-${now()}`)
+    .action(async opts => {
+      const {
+        offerId,
         previousOfferId,
+        forwardingChannel,
+        recipientAddress,
+        amount,
+        forwardingAddress,
+        ...flat
+      } = opts;
+
+      const evidence = harden({
+        aux: { forwardingChannel, recipientAddress },
+        tx: { amount, forwardingAddress },
+        ...flat,
+      });
+      mustMatch(evidence, CctpTxEvidenceShape);
+
+      /** @type {OfferSpec} */
+      const offer = {
+        id: offerId,
+        invitationSpec: {
+          source: 'continuing',
+          previousOffer: previousOfferId,
+          /** @type {string & keyof OperatorKit['invitationMakers'] } */
+          invitationMakerName: 'SubmitEvidence',
+          /** @type {Parameters<OperatorKit['invitationMakers']['SubmitEvidence']> } */
+          invitationArgs: [evidence],
+        },
+        proposal: {},
+      };
+
+      outputActionAndHint(
```

Comment:
I guess this is okay for now, would be great if it just went ahead and did the tx. There's an example here of it broadcasting a noble tx: https://github.com/Agoric/agoric-sdk/blob/master/packages/fast-usdc/src/util/noble.js#L46. This feedback is optional.

Reply:
It would be great but there's also the matter of managing key material which is already in agd.

Reply:
That's understandable, it's reasonable to assume this will be run in an environment with agd already. For noble, I didn't want to require nobled, so I added it.

Reply:
I do hope `@agoric/client-utils` will sort of standardize on the `signAndBroadcast(addr, [msg], ...)` API. It's an interface that can be implemented using either `agd` or keplr.

something like:

```js
makeAgd({execFile}).withOpts({keyringBackend: 'test'}).account('gov1').signAndBroadcast(...)
```

Reply:
Good idea. I've added that to
- https://github.com/Agoric/agoric-sdk/issues/5912